### ISSUE TITLE
remove qemu:commandline after first start (at least give a hint in documentation)

### ISSUE THREAD

**Issue by pampelmuse76** [coreos/fedora-coreos-docs#502]

**Comment by dustymabe** [comment]
Right. There's no way to make it optional. As you found out you must review it from the machine definition.

We can improve our docs around this. We'd probably need to edit https://github.com/coreos/fedora-coreos-docs/blob/main/modules/ROOT/pages/getting-started-libvirt.adoc

**Comment by pampelmuse76** [comment]
I suggest adding the following hint:

virt-xml core --edit --xml xpath.delete=./qemu:commandline
